CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
You no longer need to do that. The current active development version (unreleased in npm) is the latest versionin the CHANGELOG.

Reply:
You should add your change to 3.4.0 (next version) once you rebase.